src/components/profile/profile.tsx

```diff
@@ -14,14 +20,133 @@ const Profile = () => {
   const { showMessage } = useMessages();
   const { user, logoutUser } = useAuth();
 
+  const [userIVAs, setUserIVAs] = useState<IVA[]>([]);
+
+  const [toDeleteIVA, setToDeleteIVA] = useState<IVA | null>(null);
+  const [toConfirmIVA, setToConfirmIVA] = useState<IVA | null>(null);
+
+  const [showNewIVAModal, setShowNewIVAModal] = useState(false);
+  const [showDeletionConfirmationModal, setShowDeletionConfirmationModal] =
+    useState<boolean>(false);
+  const [showRequestVerificationModal, setShowRequestVerificationModal] =
+    useState(false);
+  const [showConfirmVerificationModal, setShowConfirmVerificationModal] =
+    useState(false);
+
+  const DeletionConfirmationModal = () => {
+    return (
+      <Modal
+        show={showDeletionConfirmationModal}
+        onHide={() => {
+          setShowDeletionConfirmationModal(false);
+        }}
+        centered
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>Confirm deletion of contact address</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          Confirm deleting the{" "}
+          <em>{toDeleteIVA ? IVAStatus[toDeleteIVA.status] + " " : ""}</em>
+          contact address <em>{toDeleteIVA?.value}</em>
+        </Modal.Body>
```

Comment:
For already verified addresses, point out the consequences with one additional sentence in the deletion confirmation dialog.

Reply:
Done 👍